packages/utilities/psammead-locales/moment/ar.js

```diff
@@ -0,0 +1,29 @@
+const moment = require('moment');
+require('moment/locale/ar');
+
+// these keys have been overridden to maintain western numerals
```

Comment:
To be explicit: `...and to ensure that double month names are used`?

Maybe we could move this to L24 for consistency with `yo.js` too.

Reply:
IMO i think its better for the comment to be on L4 unlike L24 as it describes the changes that have been over ridden which happens on the months declaration. which is more or less the same concept that is on `yo.js`

Reply:
It seems confusing to me because it mentions 'keys' here but there are no keys, whereas in the original scope there were.

Reply:
updated